README.md

```diff
@@ -12,7 +12,7 @@ binaries and pre-compiled topologies for snapshot, stable and LTS releases.
 
 # Repository Layout
```

Comment:
@dandv we usually add some labels to commit messages to quickly figure out the area the commit touches.

In your case it should be: ```README: Fix typo```

Care to update the commit message?

Reply:
Promise to do that next time :) This was just a GitHub drive-by typo fix, and I don't see any UI to change only the commit message?

Reply:
Ok, I understand.